tests/unit/changeset-test.js

```diff
@@ -90,6 +90,15 @@ test('#get returns change that is a blank value', function(assert) {
   assert.equal(result, '', 'should proxy to change');
 });
 
+test('#get with nested key returns correct value', function(assert) {
+  let date = {};
+  let foo = { date };
+  let dummyModel = EmberObject.create({ foo });
+  let changeset = new Changeset(dummyModel);
+  assert.equal(changeset.get('foo'), foo);
+  assert.equal(changeset.get('foo.date'), date);
```

Comment:
I think this should be a deep equals here

Reply:
I wanted to check for the exact object, so equal was what I was going for.

Reply:
ya, but `{} === {}` is false.

Reply:
That's not what was being tested. foo is already known. `foo === foo` would be true. It's not testing a new empty object against another empty object.

Reply:
🤦‍♂️ yes. Sorry. ... 🐑 ...